libs/ui-toolkit/src/components/key-value-table/key-value-table.test.tsx

```diff
@@ -12,7 +12,7 @@ const props: KeyValueTableProps = {
 it('Renders the correct elements', () => {
   const { container } = render(
     <KeyValueTable {...props}>
-      <KeyValueTableRow>
+      <KeyValueTableRow inline={true}>
```

Comment:
<img width="631" alt="image" src="https://user-images.githubusercontent.com/16125548/166424263-bda077e8-f6e3-4e00-bbdd-cb434b5d2385.png">
when inline is true this is how it looks like. 

I set `inline` to `false` when I have a row with json blob. Let me know if there is a preference to keep them inline.

Reply:
I think its useful to be able to pick and chose inline or not so all good. I do think if its inline though the label text should be vertically aligned to the top of the cell